index.js

```diff
@@ -118,7 +118,16 @@ module.exports = class Beanstalk {
   putJob (data) {
     return new Promise((resolve, reject) => {
       data = JSON.stringify(data)
-      this.client.put(0, 0, data._ttl || (this.defaultTTL * 2), data, function (err, jobid) {
```

Comment:
`data` shouldnt contain any "hiden" parameters for `putJob`, it was my mistake to add `_ttl`. Please remove it and add `ttl`, `priority` and `delay` as parameters for `putJob` with default values